marimo/_runtime/complete.py

```diff
@@ -12,6 +13,10 @@
 import jedi  # type: ignore # noqa: F401
 import jedi.api  # type: ignore # noqa: F401
 
+# jedi dependency
+import parso
+from parso.python.tree import PythonErrorNode, PythonNode
```

Comment:
just in case, can you push this into `_resolve_chained_key_path` and import lazily